kudos-board-api/controllers/boardController.js

```diff
@@ -1,127 +1,130 @@
-    }
-
-    // fetch boards from the database
-    const boards = await prisma.board.findMany({
-        where,
-        orderBy: orderBy.length ? orderBy : undefined,
-        include:{
-            categories:{
-                include:{
-                    category:true
-                }
-            }
-        }
-    });
-
-    res.json(boards)
-}
+  const where = {};
+  const { category, sort } = req.query;
+  const orderBy = [];
+
+  // filter by category
+  if (category) {
+    where.category = category;
+  }
+
+  // Could just check for the sort parameter and then just sort by title
+  if (sort === "title") {
+    orderBy.push({ [sort]: "asc" });
+  }
+
+  // fetch boards from the database
+  const boards = await prisma.board.findMany({
+    where,
+    orderBy: orderBy.length ? orderBy : undefined,
+    include: {
+      categories: {
+        include: {
+          category: true,
+        },
+      },
+    },
+  });
+
+  res.json(boards);
+};
 
 // gets boards by title (case-insensitive search)
-exports.getByTitle = async (req, res) =>{
-    try {
-        const title = req.params.title
-
-        const titleResults = await prisma.board.findMany({
-            where:{
-                title:{
-                    contains: title,
-                    mode:'insensitive'
-                }
-            },
-            include:{
-            categories:{
-                include:{
-                    category:true
-                }
-            }
-        }
-        })
-
-        res.json(titleResults)
-    } catch (error) {
-        console.log(error.message)
-    }
-}
+exports.getByTitle = async (req, res) => {
+  try {
+    const title = req.params.title;
+
+    const titleResults = await prisma.board.findMany({
+      where: {
+        title: {
+          contains: title,
+          mode: "insensitive",
+        },
+      },
+      include: {
+        categories: {
+          include: {
+            category: true,
+          },
+        },
+      },
+    });
 
+    res.json(titleResults);
+  } catch (error) {
+    console.log(error.message);
+  }
+};
 
 // get board by ID
 exports.getById = async (req, res) => {
-    const id = Number(req.params.id);
-    const board = await prisma.board.findUnique({where: {id},
-    include:{
-            categories:{
-                include:{
-                    category:true
-                }
-            }
-        }});
-    if(!board){
-        return res.status(404).json({error: "Not Found!"});
-    } else {
-        res.json(board);
-    }
-}
-
-
+  const id = Number(req.params.id);
+  const board = await prisma.board.findUnique({
+    where: { id },
+    include: {
+      categories: {
+        include: {
+          category: true,
+        },
+      },
+    },
+  });
+  if (!board) {
+    return res.status(404).json({ error: "Not Found!" });
+  } else {
+    res.json(board);
+  }
+};
+
+// make this more descriptive
 // post
 exports.create = async (req, res) => {
```

Comment:
You could avoid needing a comment if you make this function more descriptive by name.